### PR TITLE
postgres role

### PR DESCRIPTION
**Pull request by ovv**

### CODE REVIEW

File: roles/postgres/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  apt:
+    pkg: "{{ item }}"
+    state: latest
+    force: yes
+  with_items:
+    - postgresql-9.6
+    - postgresql-client-9.6
+    - postgresql-contrib-9.6
+    - pgtop
+    - python-psycopg2
```
Comment (mattrasband) [inline roles/postgres/tasks/main.yml:12]: I am guessing this is required for ansible to run postgres modules?

File: roles/postgres/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+
+postgres_locale: en_US.UTF-8
+postgres_roles: 'NOSUPERUSER,NOCREATEDB,NOCREATEROLE,NOLOGIN'
```
Comment (mattrasband) [inline roles/postgres/defaults/main.yml]: from a yaml perspective, may be better to make these a real list and we can use jinja to convert them to a CSV

**Review by mattrasband — APPROVED** [review]
Looks good, two minor comments - but one is just a matter of opinion and the other is just clarifying my thought.

nice @ovv!

File: roles/postgres/tasks/main.yml (hunk 1 above)
Comment (ovv) [reply roles/postgres/tasks/main.yml:12]: `python-psycopg2` yes the others are nice/must have

**Pull request merged**